Base/QTCore/qSlicerExtensionsManagerModel.h

```diff
@@ -340,6 +340,9 @@ public slots:
 
   void updateModel();
 
+  /// Number of operations in progress
+  int activeTaskCount() const;
```

Comment:
Move this outside of the public slots and instead add a property:

```
Q_PROPERTY(int activeTaskCount READ activeTaskCount)
```

To be consistent, I am wondering if this should be renamed to `numberOfActiveTasks`. I have no strong feeling about it.

Reply:
...Count is Qt style, numberOf... is VTK style. We don't use them very consistently. Since in this class the numberOf... is already used, I'll switch to that.

Reply:
> Count is Qt style, numberOf... is VTK style. We don't use them very consistently.

Good point. I added a note to https://www.slicer.org/wiki/Documentation/Nightly/Developers/Style_Guide#Naming

Reply:
To not break our own rules then, I'll make the other numberOf... methods/properties in this class deprecated and use ...Count for all. There are a couple of (not too many) other At classes that use numberOf, but for now I'll leave them alone.